config/apply/ccsm/translate.js

```diff
@@ -407,81 +399,88 @@ export function translateResponse(customApiResponse, patientData) {
-    conclusionCodes = mapResults(papResults, customCytologyCodes, standardCytologyCodes, conclusionCodes);
+      // Map the custom transformation zone results to our standard codes
+      if (transformationZone) {
+        const transformationZoneArray = [transformationZone];
+        conclusionCodes = mapResults(transformationZoneArray, customCytologyCodes, standardCytologyCodes, conclusionCodes);
+      }
+      // Map the custom HPV results to our standard codes
+      conclusionCodes = mapResults(hpvResults, customHpvCodes, standardHpvCodes, conclusionCodes);
 
-    // Map the custom transformation zone results to our standard codes
-    if (transformationZone) {
-      const transformationZoneArray = [transformationZone];
-      conclusionCodes = mapResults(transformationZoneArray, customCytologyCodes, standardCytologyCodes, conclusionCodes);
-    }
-    // Map the custom HPV results to our standard codes
-    conclusionCodes = mapResults(hpvResults, customHpvCodes, standardHpvCodes, conclusionCodes);
+      // Map the custom histology results to our standard codes
+      conclusionCodes = mapResults(colposcopyResults, customHistologyCodes, standardHistologyCodes, conclusionCodes);
+
+      // Map the custom ECC results to our standard codes
+      conclusionCodes = mapResults(endocervicalCuretageResults, customEccCodes, standardHistologyCodes, conclusionCodes);
+
+      // Map the custom excision results to our standard codes
+      if (ExcisionResultsShowAisOrCancer) {
+        conclusionCodes = mapResults(excisionResults, customExcisionCodes, standardHistologyCodes, conclusionCodes);
+      }
 
-    // Map the custom histology results to our standard codes
-    conclusionCodes = mapResults(colposcopyResults, customHistologyCodes, standardHistologyCodes, conclusionCodes);
+      // Create a DiagnosticReport resource from Order
+      const newDiagnosticReport = {
+        'resourceType': 'DiagnosticReport',
+        'id': orderId,
+        'status': 'final',
+        'subject': patient_reference,
+        'category': [
+          {
+            'coding': [
+              {
+                'system': 'http://terminology.hl7.org/CodeSystem/v2-0074',
+                'code': 'LAB'
+              }
+            ]
+          }
+        ],
+        'code': { 'coding': codings },
+        'conclusionCode': conclusionCodes,
+        'effectiveDateTime': order.ResultDate,
+        'issued': order.ResultDate
+      };
 
-    // Map the custom ECC results to our standard codes
-    conclusionCodes = mapResults(endocervicalCuretageResults, customEccCodes, standardHistologyCodes, conclusionCodes);
 
-    // Map the custom excision results to our standard codes
-    if (ExcisionResultsShowAisOrCancer) {
-      conclusionCodes = mapResults(excisionResults, customExcisionCodes, standardHistologyCodes, conclusionCodes);
+        patientData.push(newDiagnosticReport);
+        console.log('DiagnosticReport: ' + newDiagnosticReport.id);
```

Comment:
Seeing a warning about these indented incorrectly.
```suggestion
      patientData.push(newDiagnosticReport);
      console.log('DiagnosticReport: ' + newDiagnosticReport.id);
```

Reply:
Updated in https://github.com/ccsm-cds-tools/ccsm-cds-service/pull/27/commits/9e264bd967cfb6a6c9b61b151165590f3ca09118, since this is a straightforward syntactical change.